extensions/android/src/list-android.tsx

```diff
@@ -0,0 +1,88 @@
+import {
+  Action,
+  ActionPanel,
+  getApplications,
+  getPreferenceValues,
+  Icon,
+  List,
+  popToRoot,
+  showToast,
+  Toast,
+} from "@raycast/api";
+import fs from "fs";
+import { useEffect, useState } from "react";
+const { exec } = require("child_process");
+
+export default function Command() {
+  const [items, setItems] = useState<string[]>(() => []);
+  const [loading, setLoading] = useState(true);
+
+  const projectsDirectory = getPreferenceValues().androidProjectsDirectory;
+
+  useEffect(() => {
+    async function listDir() {
+      await listDirectories(projectsDirectory)
+        .then((value) => {
+          const items = value
+            .filter((dirent) => dirent.isDirectory())
+            .map((dirent) => dirent.name);
+
+          setItems(items);
+          showToast(Toast.Style.Success, "Loaded!");
+        })
+        .catch((err) => {
+          showToast(Toast.Style.Failure, "Something wrong happend!", err);
+          console.error("Error occured while reading directory!", err);
+        })
+        .finally(() => {
+          setLoading(false);
+        });
+    }
+    listDir();
+  }, []);
+
+  return (
+    <List isLoading={loading}>
+      {items?.map((project: string, index) => (
+        <List.Item
+          icon={{ source: "android-os.png" }} //TODO: load app icon
+          key={index}
+          title={project}
+          accessories={[{ icon: Icon.Folder }]}
+          actions={
+            <ActionPanel>
+              <Action
+                title="Open Project"
+                onAction={() => openProject(projectsDirectory, project)}
+              />
+            </ActionPanel>
+          }
+        />
+      ))}
+    </List>
+  );
+}
+
+function openProject(projectsDirectory: any, project: string): void {
+  return exec(
+    `open -na Android\\ Studio.app --args ${projectsDirectory}/${project}`,
```

Comment:
This could potentially be dangerous since there is no protection of params, you could use something like `spawn` instead - also you could check if `Android Studio` is installed.